docs/workers.md

```diff
@@ -52,24 +52,20 @@ synapse process.)
 
```

Comment:
Looks like these changes were supposed to be part of https://github.com/matrix-org/synapse/pull/7233?

Reply:
well, they kinda assume #7233 has landed, which makes it safe to remove `daemonize: true`, but apart from that, these changes are to split out the synctl-specific stuff from the general stuff, which I see more as part of this PR. So, no, not really.